mimic/rest/nova_api.py

```diff
@@ -315,6 +315,30 @@ def get_flavor(self, request, tenant_id, flavor_id):
         request.setResponseCode(response_data[1])
         return json.dumps(response_data[0])
 
+    @app.route('/v2/<string:tenant_id>/flavors', methods=['GET'])
+    def get_flavor_list(self, request, tenant_id):
+        """
+        Returns a list of flavor with the response code 200.
+        TO DO: The length of flavor list can be set using the control plane.
+               Also be able to set different flavor types in the future.
+        """
+        return (
+            self._region_collection_for_tenant(tenant_id)
+            .list_flavors(include_details=False, absolutize_url=self.url)
+        )
+
+    @app.route('/v2/<string:tenant_id>/flavors/details', methods=['GET'])
```

Comment:
the route on line 318 above is the /flavors route and the route on line 330 is /flavors/details.   The details route should be detail